excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -237,13 +292,34 @@ public static String getCellValue(Workbook wb, Cell cell) {
         return result;
     }
 
+    public static Object getCellValueChecked(final Workbook wb, final Cell cell, final ColumnType expectedColumnType) {
```

Comment:
Can you make this method private and rename the `wb` parameter to `workbook`?

Reply:
done